test/unit/eigensolver/test_tridiag_solver.cpp

```diff
@@ -158,7 +161,7 @@ void solveRandomTridiagMatrix(SizeType n, SizeType nb) {
   std::vector<RealParam> diag_arr(to_sizet(n));
   std::vector<RealParam> offdiag_arr(to_sizet(n));
   SizeType diag_seed = n;
-  SizeType offdiag_seed = n - 1;
+  SizeType offdiag_seed = n + 1;
```

Comment:
Hmm, why is this needed?

Reply:
Because `n == 0` make `offdiag_seed = -1`, which will internally get casted to `size_t` (more generally an `unsigned int`) as it is the input seed for the random engine `std::mt19937_64`.